airbyte-integrations/connectors/source-pagarme/source_pagarme/streams.py

```diff
@@ -0,0 +1,170 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import datetime
+from abc import ABC
+from typing import Any, Iterable, Mapping, MutableMapping, Optional
+
+import requests
+from airbyte_cdk.sources.streams.http import HttpStream
+
+
+class PagarmeStream(HttpStream, ABC):
+    page = 1
+    page_size_limit = 1000
```

Comment:
What do you think about adding a new config property to configure this `page_size_limit`?